addon/webextension/selector/ui.js

```diff
@@ -275,14 +277,16 @@ this.ui = (function() { // eslint-disable-line no-unused-vars
     },
 
     hide() {
-      window.removeEventListener("scroll", this.onScroll);
+      window.removeEventListener("scroll", watchFunction(assertIsTrusted(this.onScroll)));
```

Comment:
I didn't catch this before, but just as with `this.onResize`, if we wrap `this.onScroll` inline, removeEventListener will fail to detach the listener. It doesn't lead to any thrown errors, but is a memory leak. I'm fine to address this here or in a followup bug.